apps/web/src/pages/onboarding/ui/index.tsx

```diff
@@ -1,9 +1,32 @@
-import { OnboardingForm } from '@/widgets/onboarding-form';
+import { GithubLogin } from '@/features/github-login';
 
 export function OnboardingPage() {
   return (
-    <div className="flex items-center justify-center h-full">
-      <OnboardingForm />
+    <div className="relative flex items-center justify-center h-screen bg-gray-950 overflow-hidden">
+      <div className="absolute top-[8%] left-[15%] w-[400px] h-[250px] bg-blue-800 rounded-[60%] blur-[190px] opacity-14"></div>
+      <div className="absolute top-[40%] right-[5%] w-[450px] h-[300px] bg-purple-800 rounded-[50%] blur-[210px] opacity-18"></div>
+      <div className="absolute bottom-[30%] left-[5%] w-[420px] h-[320px] bg-indigo-800 rounded-[45%] blur-[200px] opacity-15"></div>
+      <div className="absolute bottom-[15%] right-[25%] w-[380px] h-[230px] bg-teal-800 rounded-[55%] blur-[220px] opacity-12"></div>
+      <div className="absolute top-[55%] left-[40%] w-[500px] h-[300px] bg-gray-800 rounded-[50%] blur-[240px] opacity-10"></div>
```

Comment:
셀프 클로징으로 처리하는게 좋을 것 같습니다!

```suggestion
      <div className="absolute top-[8%] left-[15%] w-[400px] h-[250px] bg-blue-800 rounded-[60%] blur-[190px] opacity-14" />
      <div className="absolute top-[40%] right-[5%] w-[450px] h-[300px] bg-purple-800 rounded-[50%] blur-[210px] opacity-18" />
      <div className="absolute bottom-[30%] left-[5%] w-[420px] h-[320px] bg-indigo-800 rounded-[45%] blur-[200px] opacity-15"></div>
      <div className="absolute bottom-[15%] right-[25%] w-[380px] h-[230px] bg-teal-800 rounded-[55%] blur-[220px] opacity-12" />
      <div className="absolute top-[55%] left-[40%] w-[500px] h-[300px] bg-gray-800 rounded-[50%] blur-[240px] opacity-10" />
```